docs/tutorials/validate_data.rst

```diff
@@ -0,0 +1,447 @@
+.. _tutorial_validate_data:
+
+
+
+Validate Data
+==============
+
+Expectations describe data assets. Data assets are composed of batches. Validation checks expectations against a batch of data.
+
+Validation = checking if a batch of data from a data asset X conforms to all expectations in expectation suite Y. Expectation suite Y is a collection of expectations that you created that specify what a valid batch of data asset X should look like.
+
+To run validation you need a **batch** of data. To get a **batch** of data you need:
+
+* to specify which **data asset** the batch is from
+* to specify an **expectation suite** to validate against
+
+
+This tutorial will explain each of these objects, show how to obtain them, execute validation and view its result.
```

Comment:
Wow. <3

Reply:
@jcampbell - agree - this is @Aylr 's suggestion from yesterday!